pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -1016,7 +1016,11 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
 
         // Does the player own the airport?
         if (endid != playerid)
-            return 0;
+        {
+            SendClientMessage(playerid, COLOR_ORANGE, "Error: Sorry, You Don't Own The LV Airport"); //Fix "Server: Unkown Command" issue
+            SendClientMessage(playerid, Color::Green, "use /taxi 2 to be teleported to LV Airport");
+        }
+        return 0;
```

Comment:
So, this is the bit that actually causes the "Server: Unknown command" message— returning `1` means that the command could be handled, returning `0` means that it couldn't.

In this case, the player will receive a message about not owning the airport, so the command has been handled. This means that you should return `1`.